include/tracer.hrl

```diff
@@ -0,0 +1,47 @@
+%% macros for tracing
+%% register a tracer for an application with opentelemetry:register_application_tracer(AppName)
+
+-define(current_tracer, opentelemetry:get_tracer(?MODULE)).
+-define(current_span_ctx, ot_tracer:current_span_ctx(?current_tracer)).
+
+-define(start_span(SpanName),
+        ot_tracer:start_span(?current_tracer, SpanName, #{})).
+
+-define(start_span(SpanName, Opts),
+        ot_tracer:start_span(?current_tracer, SpanName, Opts)).
+
+-define(set_span(SpanCtx),
+        ot_tracer:set_span(?current_tracer, SpanCtx)).
+
+-define(with_span(SpanName, Attributes, Fun),
+        ot_tracer:with_span(?current_tracer, SpanName, Fun)).
+
+-define(end_span(),
+        ot_tracer:end_span(?current_tracer)).
+
+-define(current_span_ctx(),
+        ot_tracer:current_span_ctx(?current_tracer)).
+
+-define(is_recording_events(),
+        ot_span:is_recording_events(?current_tracer, ?current_span_ctx)).
```

Comment:
Yea, not sure. I hadn't taken the time to look and just added it because maybe it was useful. I'll check and remove it unless I can find reason to keep it.